src/Native/Windows/CMakeLists.txt

```diff
@@ -97,8 +94,6 @@ endif()
 
 if (__LinkCustomCRT STREQUAL "true" AND $ENV{__BuildArch} STREQUAL "arm" )
     set(__LinkArgs "${__LinkArgs} /DEFAULTLIB:ole32.lib")
-elseif(__LinkCustomCRT STREQUAL "true" AND NOT $ENV{__BuildArch} STREQUAL "arm" )
```

Comment:
Why is this not needed anymore?

Reply:
TL;DR: Now that we're packaging api sets in the runtime package, we don't need to link a custom crt here anymore

The "__LinkCustomCRT" property was added to the AppX build so we could build a clrcompression without dependencies on the CRT api sets. This was specifically to accommodate machines without the CRT available that were using AOT clrcompression builds. The idea was to use this as a stop-gap solution until Reverse Forwarders came in... RS2 I think it was? However, we appear to now be packaging the necessary crt api set dlls in the runtime package anyways, so we can just use those and not bother with the custom crt code.

The reason this actual line was removed was because this old custom crt of course doesn't provide all the functions that I needed for Brotli, so I needed to open up the default libs. Really I can remove all of the "__LinkCustomCRT" stuff, but I wanted to keep the delta here small.

There was some contention in the 1.0 timeframe around clrcompression and other native binaries requiring the target machine to have the crt api sets available, so that was the reasoning for this particular workaround (which was actually ripped from one of our netfx builds). If I remember correctly there was an ownership issue with statically linking what we needed because it put onto us the onus of servicing those dependencies. It appears that the long-term resolution for this in coreclr was to just package the api sets for the crt in the runtime package so we can dynamically link with abandon, but I've been off the radar on this for a while and missed the discussion on that. I imagine the cost of servicing/ownership was far lower than the cost of working around missing API sets on the target machine.

Reply:
Though now that I think of it, since we're not including brotli for Uap right now, we don't need to change this as a part of this PR.

Reply:
>  I can remove all of the "__LinkCustomCRT" stuff

Yes, I think that would be better than deleting just part of it.